schemas/core/modes/MODE_SCOOTER.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "http://maasglobal.com/core/modes/MODE_SCOOTER.json",
+  "type": "object",
+  "properties": {
+    "pickupStation": {
```

Comment:
I suggest adding properties already existed by API and add more on the go rather than copy pasting from another similar mode if it is the attention.
If not, ignore what i said :)

Reply:
good point